addons/stock/models/stock_move.py

```diff
@@ -168,7 +168,11 @@ def _default_group_id(self):
     @api.onchange('product_id', 'picking_type_id')
     def onchange_product(self):
         if self.product_id:
-            self.description_picking = self.product_id._get_description(self.picking_type_id)
+            if self.picking_id:
+                product = self.product_id.with_context(lang=self.picking_id.partner_id.lang or self.env.user.lang)
```

Comment:
Actually, if there is no picking, `self.picking_id.partner_id.lang` will be `False`. So this should be enough in all cases:
```
product = self.product_id.with_context(lang=self.picking_id.partner_id.lang or self.env.user.lang)
```

Reply:
Thought I'd avoid unnecessary calls... Done :+1: